# fix "invalid suffix on literal; C++11 requires a space between literal and identifier"
--- superpose.cpp.orig	2026-01-29 14:46:28
+++ superpose.cpp	2026-01-29 14:57:37
@@ -114,7 +114,7 @@
 #endif
     " Protein Structure Superposition\n"
     " -------------------------------\n"
-    " v."superpose_version" from "superpose_date" built with SSM v.%i.%i.%i, "
+    " v.%s from %s built with SSM v.%i.%i.%i, "
     "MMDB v.%i.%i.%i\n"
     "\n"
     " USAGE:\n"
@@ -140,7 +140,7 @@
 #ifdef _ccp4_
     "$$\n"
 #endif
-    ,ssm::MAJOR_VERSION,ssm::MINOR_VERSION,ssm::MICRO_VERSION,
+    ,ssm::MAJOR_VERSION,superpose_version,superpose_date,ssm::MINOR_VERSION,ssm::MICRO_VERSION,
      mmdb::MAJOR_VERSION,mmdb::MINOR_VERSION,mmdb::MICRO_VERSION,
      argv0,argv0,argv0
    );
@@ -295,9 +295,9 @@
   }

   printf ( "\n"
-    " Superpose v."superpose_version" from "superpose_date" "
+    " Superpose v.%s from %s "
                                         "(based on SSM algorithm)\n"
-    " ---------------------------------------------------------\n\n"
+    " ---------------------------------------------------------\n\n",superpose_version,superpose_date
    );

   mmdb::InitMatType();
